{"success": false, "error": "Traceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 544, in _run_solc\n    ret: Dict = json.loads(stdout)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/__init__.py\", line 357, in loads\n    return _default_decoder.decode(s)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/json/decoder.py\", line 355, in raw_decode\n    raise JSONDecodeError(\"Expecting value\", s, err.value) from None\njson.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 826, in main_impl\n    ) = process_all(filename, args, detector_classes, printer_classes)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/slither/__main__.py\", line 86, in process_all\n    compilations = compile_all(target, **vars(args))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 637, in compile_all\n    compilations.append(CryticCompile(target, **kwargs))\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 117, in __init__\n    self._compile(**kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/crytic_compile.py\", line 548, in _compile\n    self._platform.compile(self, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 154, in compile\n    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 276, in _get_targets_json\n    return _run_solc(\n  File \"/Users/giacomoibba/miniconda3/lib/python3.8/site-packages/crytic_compile/platform/solc.py\", line 548, in _run_solc\n    raise InvalidCompilation(f\"Invalid solc compilation {stderr}\")\ncrytic_compile.platform.exceptions.InvalidCompilation: Invalid solc compilation Error: Multiple SPDX license identifiers found in source file. Use \"AND\" or \"OR\" to combine multiple licenses. Please see https://spdx.org for more information.\n--> /Users/giacomoibba/Desktop/mainnet/2a/2acdE0f8A8998615Dd926A5Ac7E79135832052d2_LSDUpdateBalance.sol\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/LSDBase.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/mainnet/2a/2acdE0f8A8998615Dd926A5Ac7E79135832052d2_LSDUpdateBalance.sol:4:1:\n  |\n4 | import \"../LSDBase.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/interface/owner/ILSDOwner.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/mainnet/2a/2acdE0f8A8998615Dd926A5Ac7E79135832052d2_LSDUpdateBalance.sol:6:1:\n  |\n6 | import \"../../interface/owner/ILSDOwner.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/interface/balance/ILSDUpdateBalance.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/mainnet/2a/2acdE0f8A8998615Dd926A5Ac7E79135832052d2_LSDUpdateBalance.sol:7:1:\n  |\n7 | import \"../../interface/balance/ILSDUpdateBalance.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/interface/token/ILSDTokenLSETH.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/mainnet/2a/2acdE0f8A8998615Dd926A5Ac7E79135832052d2_LSDUpdateBalance.sol:8:1:\n  |\n8 | import \"../../interface/token/ILSDTokenLSETH.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/interface/token/ILSDTokenVELSD.sol\" not found: File not found.\n --> /Users/giacomoibba/Desktop/mainnet/2a/2acdE0f8A8998615Dd926A5Ac7E79135832052d2_LSDUpdateBalance.sol:9:1:\n  |\n9 | import \"../../interface/token/ILSDTokenVELSD.sol\";\n  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/interface/vault/ILSDRPVault.sol\" not found: File not found.\n  --> /Users/giacomoibba/Desktop/mainnet/2a/2acdE0f8A8998615Dd926A5Ac7E79135832052d2_LSDUpdateBalance.sol:10:1:\n   |\n10 | import \"../../interface/vault/ILSDRPVault.sol\";\n   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/token/ERC20/IERC20.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2acdE0f8A8998615Dd926A5Ac7E79135832052d2_LSDUpdateBalance.sol:139:1:\n    |\n139 | import \"@openzeppelin/contracts/token/ERC20/IERC20.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"@openzeppelin/contracts/token/ERC20/IERC20.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2acdE0f8A8998615Dd926A5Ac7E79135832052d2_LSDUpdateBalance.sol:175:1:\n    |\n175 | import \"@openzeppelin/contracts/token/ERC20/IERC20.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\nError: Source \"/Users/giacomoibba/Desktop/mainnet/interface/ILSDStorage.sol\" not found: File not found.\n   --> /Users/giacomoibba/Desktop/mainnet/2a/2acdE0f8A8998615Dd926A5Ac7E79135832052d2_LSDUpdateBalance.sol:268:1:\n    |\n268 | import \"../interface/ILSDStorage.sol\";\n    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\n\n", "results": {}}
